css/index.css

```diff
@@ -143,4 +144,175 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+.header {
+    display: flex;
+    margin-top: 10px;
+    margin-bottom: 30px;
+    align-items: baseline;
+}
+.nav {
+    display: flex;
+    justify-content: space-evenly;
+    width: 800px;
+}
+.navie {
+    margin-right: 30px;
+    margin-left: 20px;
+}
+.navie:hover
+{   color: red;
+    animation: pop 0.3s linear 1;
+  }
+  @keyframes pulse{
+    25%  {transform: scale(0.9);}
+    75%  {transform: scale(1.1);}
+  }
+  @keyframes push{
+    50%  {transform: scale(0.8);}
+  }
+  @keyframes pop{
+    50%  {transform: scale(1.2);}
+  }
+  .middle-content .boxes .box2 {
```

Comment:
Good thinking to change the class name so you can treat each of these boxes separately. What I would have done is this:

```
<div class="box box-1">Box 1</div>
```

That way on the box class, you can write the code that dictates the height, width, etc...all the stuff that is the same amongst all the boxes.

Then use the individual class name box-1 to override the background color on the box.

So the CSS would look something like this :
```
.box {
    height: 100px;
    width: 100px;
    display: flex;
    justify-content: center;
    align-items: center;
    color: white;
}

.box-1 {
    background-color: teal;
}

```

That way you don't have to keep writing that code over and over again.